scripts/mapq0_cnv.py

```diff
@@ -0,0 +1,90 @@
+import subprocess
+import argparse
+import tempfile
+import os.path
+import pysam
+import pysamstats
+import numpy as np
+import sys
+
+#checks if file exists
+def is_valid_file(parser, arg):
+    if not os.path.exists(arg):
+        parser.error("The file {} does not exist!".format(arg))
+    else:
+        return open(arg, 'r')  # return an open file handle
+
+#takes in bam file and distance for merge function and outputs bed file showing regions with map qual 0
+def merge(bam_file, distance, cutoff, output):
+	tf = tempfile.NamedTemporaryFile(delete=False, suffix='.bam')
```

Comment:
Why delete=False? Don't we want to clean it up afterwards? You might want to use a with statement as shown in the examples: https://docs.python.org/3/library/tempfile.html#examples

Reply:
Oh, good idea! Without delete=False the file was disappearing before I used it, but the issue doesn't occur if I do a with statement so I will do that! Thanks for the suggestion.